js/plugin/services.ts

```diff
@@ -99,7 +99,6 @@ export const ServicesMixin = (SuperClass) => {
           for (const [k, v] of Object.entries<any>(d)) {
             if (k.endsWith("_action")) {
               let actions = v; // Force Closure. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Closures#creating_closures_in_loops_a_common_mistake
-              let key = k; // If required use key in anonymous function to avoid closure issue as per above comment
```

Comment:
Can we leave as a comment in case anyone ever wishes to use `k` that they are aware of closure issue. Thanks.